### ISSUE TITLE
[FEATURE] Add Events for `native.keyboardhidden` & `native.keyboardvisible`

### ISSUE THREAD

**Issue by runspired** [driftyco/ionic-plugin-keyboard#62]
Opening and closing the keyboard can seriously affect app performance, and it's often best to wait until after the keyboard animation is complete to perform some tasks.

**Comment by tlancina** [comment]
Yeah right now iOS uses UIKeyboardWillHide and UIKeyboardWillShow and Android waits for the View to resize, so they technically aren't even the same thing. Definitely will be adding this for iOS, we'll see about Android.

**Comment by Hirbod** [comment]
ping

**Comment by runspired** [comment]
@tlancina still on your radar?

**Comment by tlancina** [comment]
Yeah not forgotten, I'd like to update the method naming to be consistent as well, but unfortunately have had to prioritize other projects. PRs welcome on this!
